### DISCUSSION TITLE
A very stupid question -- how to choose Yes/No when creating next app?

### DISCUSSION THREAD

**Discussion by hbhutta** [vercel/next.js#57617] [Help] [answered]
### Summary

I've tried switching with tab and the arrow keys but it doesn't work. When I type "yes" it doesn't work either. Sorry this is a very stupid question. I'm new to web development. Can someone explain?

**Comment by icyJoseph** [comment]
Moving with the arrows should work. What terminal are you using?

**Reply by hbhutta** [reply, marked as answer]
Hi! I was using Git Bash, and it didn’t work there but when I used Powershell it worked.

**Reply by waqaskhan092** [reply]
> Moving with the arrows should work. What terminal are you using?
Thanks, its really working.
